migration_challenge_keras_image/Local Notebook.ipynb

```diff
@@ -411,13 +411,8 @@
     "# The export folder needs to be empty, or non-existent\n",
     "!rm -rf data/model/model/1\n",
     "\n",
-    "sess = K.get_session()\n",
-    "tf.saved_model.simple_save(\n",
-    "    sess,\n",
-    "    os.path.join(\"data/model\", \"model/1\"),\n",
-    "    inputs={ \"inputs\": model.input },\n",
-    "    outputs={ t.name: t for t in model.outputs },\n",
-    ")"
+    "# Please ignore Layer.updates(...) warning if any while running the notebook in < TFv2.4\n",
+    "model.save(os.path.join(\"data/model\", \"model/1\"))"
```

Comment:
Is this the only change necessary to get the reference solution to work as well, over on the solution branch? I can replicate it myself if so, but if there's more complex stuff would probably need a separate PR to `solution` also

Reply:
Hi Alex, yes. I've just pushed code changes:
 * revert model training on GPU instance 
 * update the training script template for image ML problem. (this one should be bundled with migration_challenge_keras_image/Local Notebook.ipynb change.)

Let me know if it's ok to accept the PR. or I shall revert the code change on image ML problem and leave it to you. Cheers.